src/legacy/components/Popups/TransactionPopup.tsx

```diff
@@ -40,7 +42,7 @@ export function TransactionPopup({
         ) : (
           summary
         )}
-        {chainId && <ExplorerLink id={hash} type="transaction" />}
+        {chainId && <ExplorerLink id={hash} isComposableOrder={isComposableOrder} type="transaction" />}
```

Comment:
Here for example you could have instead (not settled on the name, could be `safe` or `safe-transaction` etc):

```suggestion
        {chainId && <ExplorerLink id={hash} type="safe" />}
```

Reply:
Done